src/components/navigation/Navbar.tsx

```diff
@@ -1,6 +1,6 @@
 import Link from 'next/link';
-import React, { FC, useState } from 'react';
-import logo from '../../assets/media/icons/Jupiter.png';
+import { type FC, useState } from 'react';
+import logo from '../../../assets/media/icons/Jupiter.jpg';
```

Comment:
What is the reason to switch to jpg here?

Reply:
no particular reason, I just changed it on my local branch